client/src/components/ControlPanel.vue

```diff
@@ -288,6 +298,29 @@ export default {
                   Save Note
                 </v-col>
               </v-row>
+              <v-flex
+                class="d-flex ml-5"
+                style="flex-direction:column"
+              >
+                <div style="flex-grow: 1">
+                  <v-switch
+                    :input-value="showCrosshairs"
+                    @change="setShowCrosshairs"
+                    label="Display crosshairs"
+                    hide-details
+                    class="shrink ma-0 pa-0 ml-n2"
```

Comment:
These margings are small enough to impact usability:
![image](https://user-images.githubusercontent.com/577946/152816690-e74d3142-6525-4d21-aef5-65f7ab0c0847.png)
The halo around the selected toggle is clickable, so once you have clicked a toggle, it overlaps the other toggle.

Reply:
Changed in [4bd3d76](https://github.com/OpenImaging/miqa/pull/302/commits/4bd3d76a8a3e3b2464c53e951ffd12e659add88f)